.github/workflows/stale.yaml

```diff
@@ -0,0 +1,27 @@
+name: "Close stale issues"
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: >
+          This issue has been automatically marked as stale because it
+          has not had recent activity. It will be closed if no further
+          activity occurs in the next 30 days.
+        stale-pr-message: >
+          This pull request has been automatically marked as stale because it
+          has not had recent activity. It will be closed if no further
+          activity occurs in the next 30 days.
+        days-before-stale: 90
+        days-before-close: 30
+        stale-issue-label: 'status:stale'
+        exempt-issue-labels: 'bug,confirmed,status:planned'
```

Comment:
About a third of all open issues are marked as "bug", I wouldn't exempt this label.

Reply:
I removed "confirmed" since it's not something we use anymore. I checked and as of now "bug" are 292 issues out of 1102 - I would say it's a number that I'm confident we can go through and eventually close or relabel manually.

Reply:
I also think it would be a good idea, before merging this PR, to create a small group of maintainers (4 or 5 people at most) and ping the group in the messages added by the stale bot. That will give us the opportunity to monitor more closely how the bot works.